introduction/benefits.md

```diff
@@ -0,0 +1,30 @@
+
+# What Are the Benefits of InnerSource?
+
+There are many benefits to collaborating via inner source.
+Inner source gives a company a scalable strategy for **guest teams to get feature requests when they need them** without the long-term burden of maintainance.
+The company as-a-whole wins as the time of guest teams is put into code that others can use.
+
+While that result is a shining benefit of inner source, there are many benefits to hosts that receive regular inner source contributions.
+Recall that, as part of the inner source process, the product owner on the host team agrees from the outset that contributed features are good and desirable.
+Inner source allows the host team to receive **help in creating a better product** for its consumers!
+
+**Inner sourcing provides the host team a scalable strategy** for fulfilling the varying amounts of features requested from its many consumers.
+Given the fixed capacity of the full-time members of the host team, it's likely that, at times, the combined business roadmaps of its consumers will require very (or even unreasonably) large amounts of work to be done in the host team's products.
+Without inner source, this situation easily leads to a stressed, overworked team dealing with many feature requests escalated to its leaders.
+
+However, if the host team operates via inner source, the engineering resources required to build those features will appear in-proportion to their importance in the form of guest contributors.
+**Inner source becomes a force multiplier** that allows a host team to temporarily act larger than its actual size during times of high demand.
+When the demand has ended, the team throughput scales back to normal levels, all without any micromanagement of team headcount or work items.
+
+Beyond the raw work that the host team is able to accomplish in its system, regular inner source contributions give the host team **better requirements and prioritization alignment with all of its consumers**.
+A host team can do its best requirements gathering on the work it produces, but when the consumer itself is the one submitting the work the chances are much higher that the resulting change is aligned to just what the consumer needs.
+While it may be only one single guest team submitting the change, that team is likely representative of many other consumers.
+
+In addition to this alignment, there is also a general training and education of contributors as they work with and learn from trusted committers.
+Having made a code contribution, they feel a stake in the host team project.
+It's something that they recommend to their colleagues or new teams that they join.
+They understand the project better and are able to answer questions about it to others, relieving the host team of some of some of that burden.
+This learning and cross-team alignment over time serves to **break down traditional company silos**.
```

Comment:
These are all great points to share.  As I worked on incorporating them I struggled with the fact that this document is in narrative and paragraph form, rather than a list of bullets.  This writing structure requires that content be group-able in paragraph-sized chunks that have something of a common theme.  My task, then, is to take these bullets and group them in that way.

My result doesn't incorporate every bullet directly, but I think that there is some hint of each of these ideas in the result as follows:

> more innovation through more diversity in the teams

"More people contributing to a project naturally cross-pollinates ideas"

> better documentation

"Project documentation improves"

> higher job satisfaction

"higher job satisfaction"

> increased career opportunities

"grow in their career"

> higher level of commitment

"Contributors feel a stake"

> Continued learning and professional development via exposure to new ideas and solutions
 
"This interaction helps contributors to learn and grow"

Reply:
LGTM, @rrrutledge . Thanks for incorporating the suggestions.

Reply:
🎉